R/from_sec_to_hms.R

```diff
@@ -0,0 +1,34 @@
+#' Change format to hours minutes seconds
+#' 
+#' Change format form number of seconds to number of hours, minutes and seconds
+#'
+#' @param x An integer vector. Number of seconds
+#'
+#' @return A character vector with format "X h Y min Z s", where Y < 60 and Z < 60
+#' 
+#' @export
+#'
+#' @examples
+#' from_sec_to_hms(60)
+#' from_sec_to_hms(134)
+#' from_sec_to_hms(1257)
+
+
+from_sec_to_hms <- function(x) {
+  
+  single <- function(x) {
```

Comment:
nie wiem, czy ta funkcja jest często wywoływana, ale single lepiej mieć poza nią, żeby jej nie tworzyć wiele razy

Reply:
a za to móc testować